gatling/user-files/utils/requestMIds.js

```diff
@@ -0,0 +1,28 @@
+var rp = require('request-promise');
+const fs = require('fs');
+const args = process.argv.slice(2);
+let i = parseInt(args[0]);
+
+var options = {
+    method: 'POST',
```

Comment:
Your indentation rules seem to be different than our team established ones (2 spaces, no tabs)
Please use eslint to enforce the same coding rules than in all ou repos